experimental/deploy_cycle_slurm_ndv4/scripts/gpu_monitoring_env.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+log_analytcs_ws_customer_id=$1
+log_analytcs_ws_key=$2
+
+ENV_FILE=/etc/profile.d/gpu_monitoring.sh
```

Comment:
Where does the gpu_monitoring.sh file come from?

Reply:
Its generated in gpu_monitoring_env.sh